### ISSUE TITLE
Incompatible with latest Barryvdh\Debugbar

### ISSUE THREAD

**Issue by acasar** [scottbedard/oc-debugbar-plugin#59]
When using the latest Barryvdh\Debugbar (version 3.10.4) I get the following error:

```
Class "Barryvdh\Debugbar\DataCollector\ModelsCollector" not found
```

Downgrading to 3.10.2 temporarily fixed the issue.

**Comment by barryvdh** [comment]
Ah hmm yes I think we removed that in favor of the countable one.
https://github.com/barryvdh/laravel-debugbar/pull/1517/files
Probably https://github.com/rainlab/debugbar-plugin/blob/master/datacollectors/OctoberModelsCollector.php could be modified the same.

**Comment by idpsycho** [comment]
for anyone, quick fix is to download 3.10.1

(or 3.10.2 might work too), because in 3.10.3 the modelscollector was deleted
also i had to run "php artisan optimize" to make it work
for some reason changing rainlab debugbar's composer.json dependency to 3.10.1 still installed latest vendor laravel debugbar

**Comment by barryvdh** [comment]
I'll add them back for now https://github.com/barryvdh/laravel-debugbar/pull/1531

**Comment by barryvdh** [comment]
Check with the latest release please; https://github.com/barryvdh/laravel-debugbar/releases/tag/v3.10.5

**Comment by acasar** [comment]
@barryvdh Thanks, I can confirm the issue is fixed with the latest release.

I will leave the issue open until @daftspunk has a chance to check the PR.

**Comment by daftspunk** [comment]
Tested and should be fixed in v3.3.6

**Comment by acasar** [comment]
Thanks everyone! It's working perfectly now.
